views/templates/admin/tabs/content/config.tpl

```diff
@@ -28,9 +28,29 @@
         {l s='Reassurance Block' mod='psreassurance'}
     </div>
     {include file="./config_elements/reassurance_block.tpl"}
-    {foreach from=$allblock item=$aBlock key=$key}
-        <form class="form_{$aBlock['id_psreassurance']}" method="post" ENCTYPE="multipart/form-data">
-            <div class="panel-body-{$aBlock['id_psreassurance']} show-rea-block inactive">
+    <form class="form_" method="post" ENCTYPE="multipart/form-data">
+      <div class="panel-body- show-rea-block inactive">
```

Comment:
`panel-body-` => is it an issue or is it expected ? looks like something is missing after the `-`

Reply:
Fixed